docusaurus.config.js

```diff
@@ -73,18 +87,18 @@ module.exports = {
               label: 'About',
               to: '/docs/about',
             },
-            {
-              label: 'Getting Started',
-              to: '/docs',
-            },
-            {
-              label: 'Help',
-              to: '/docs/help',
-            },
-            {
-              label: 'Vault',
-              to: '/docs/vault',
-            },
+            // {
+              // label: 'Getting Started',
+              // to: '/docs',
+            // },
+            // {
+              // label: 'Help',
+              // to: '/docs/help',
+            // },
+            // {
+              // label: 'Vault',
+              // to: '/docs/vault',
+            // },
```

Comment:
I commented this out, but it might make sense to just remove these? Although, _Getting Started_ still exists as a page that can be navigated to via the Navigation Bar.